### PR TITLE
[TE] frontend - harleyjj/validation - surface errors in dom for creat…

### PR DESCRIPTION
**Pull request by harleyjj**
…e and edit alert

1) Surfaces errors in configuration and preview to the DOM
2) Includes modal for 'more details' if response includes 'more-info' field
3) Styling updates to Create Alert and Edit Alert
4) On successful alert creation, Create Alert transitions to Alert Overview for detection
5) Fixes bug where successful Create Alert shown as error

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/template.hbs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="row te-modal__settings">
+  <div class="col-md-10 no-padding">
+    <label for="select-metric" class="control-label te-label te-label--taller required">
+      Need support? Share the content below with the Thirdeye team at
+      <a class="thirdeye-link-secondary-grey" target="_blank" href="mailto:ask_thirdeye@linkedin.com">ask_thirdeye@linkedin.com</a>
```
Comment (akshayrai) [inline thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/template.hbs]: Can you configure and fetch this email (ask_thirdeye) from the environment.js file? That way, open source users also can customize this as per their help alias.

File: thirdeye/thirdeye-frontend/app/pods/components/subscription-yaml/template.hbs [hunk 2]
```diff
@@ -46,3 +68,8 @@
     mode="ace/mode/yaml"
   }}
 </div>
+<div class="col-xs-12">
+  <label for="select-metric" class="control-label te-label te-label--taller required">Can't find your team? Contact
+    <a class="thirdeye-link-secondary-grey" target="_blank" href="mailto:ask_thirdeye@linkedin.com">ask_thirdeye@linkedin.com</a>
```
Comment (akshayrai) [inline thirdeye/thirdeye-frontend/app/pods/components/subscription-yaml/template.hbs]: Here as well - make email configurable

File: thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/component.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/**
```
Comment (akshayrai) [inline thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/component.js]: Add license header

Comment (harleyjj) [reply thirdeye/thirdeye-frontend/app/pods/components/modals/error-modal/component.js]: I added a ticket to the backlog for adding license headers to the frontend.

**Pull request merged**